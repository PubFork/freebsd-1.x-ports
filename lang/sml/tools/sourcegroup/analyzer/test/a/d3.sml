signature A = sig
  structure B :sig val x:int end
end
