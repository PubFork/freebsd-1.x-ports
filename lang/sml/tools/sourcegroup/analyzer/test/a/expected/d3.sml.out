source sml "a/d3.sml"
  export signature A ;

